build/deploy/schema-manager.libsonnet

```diff
@@ -1,5 +1,8 @@
 local base = import 'base.libsonnet';
 local volumes = import 'volumes.libsonnet';
+
+# Deprecated: Mounting schemas is required for dss images with version below 0.6.0.
```

Comment:
I think we can go ahead and remove these mounts directly -- if someone wants to deploy (e.g.) v0.5.0, my expectation would be that they would use the deployment tools in v0.5.0 of the repository, which would mount the schemas.  I don't think we need to support the use of v0.5.0 deployment tools to deploy v0.6.0 of the DSS image, which is what I think this deprecation notice + retaining the mounts is accomplishing.

Reply:
Sounds good. done.